Neptune.Database/dbo/Views/dbo.vFieldVisitDetailed.sql

```diff
@@ -36,6 +37,13 @@ left join
 left join dbo.TreatmentBMPAssessment tbasInit on fv.FieldVisitID = tbasInit.FieldVisitID and tbasInit.TreatmentBMPAssessmentTypeID = 1 -- Initial
 left join dbo.TreatmentBMPAssessment tbasPM on fv.FieldVisitID = tbasPM.FieldVisitID and tbasPM.TreatmentBMPAssessmentTypeID = 2 -- Post-Maintenance
 left join dbo.MaintenanceRecord mr on fv.FieldVisitID = mr.FieldVisitID
+left join

```

Comment:
should not be having to go down WQMPVerify to see which wqmp a treatment bmp belongs to.  Should be just left join dbo.WaterQualityManagementPlan wqmp on tb,WaterQualityManagementPlanID = wqmp.WaterQualityManagementPlanID